src/locales/en/common.json

```diff
@@ -878,6 +880,8 @@
       "title": "Network fees",
       "validate": "Confirm",
       "required": "Fees are required",
+      "chooseGas": "Choose a gas price",
+      "higherFaster": "Higher gas price means a faster confirmation",
```

Comment:
@dasilvarosa please confirm. thanks! (maybe you already did on slack, just to be sure :p )

Reply:
on slack it seems that the wording did come from @dasilvarosa